mappings/net/minecraft/Registries.mapping

```diff
@@ -0,0 +1,69 @@
+CLASS net/minecraft/class_7923 net/minecraft/util/registry/Registries
```

Comment:
This file should be moved to `mappings/net/minecraft/util/registry/Registries.mapping`. I highly recommend you use Enigma when editing mappings.